### PR TITLE
feat(STONE-855): making alerts less sensitive

### PR DESCRIPTION
**Pull request by jsztuka**
Increasing alert **for** value to make trigering of alerts less sensitive.

### CODE REVIEW

**Comment by dirgim** [comment]
I think these changes will also require updates to unit tests at https://github.com/redhat-appstudio/o11y/tree/main/test/promql/tests/data_plane

File: rhobs/alerting/data_plane/prometheus.latency_pac_provision_alerts.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ spec:
           ) / ignoring(le)
           increase(redhat_appstudio_buildservice_PaC_configuration_time_bucket{le="+Inf"}[5m])
         ) > 0.05
-      for: 1m
+      for: 10m
```
Comment (dirgim) [inline rhobs/alerting/data_plane/prometheus.latency_pac_provision_alerts.yaml:22]: I don't think this is an alert owned by the integration service, if we want to update it, we should probably notify the build service team about it.

**Comment by jsztuka** [comment]
@mmorhun 
https://github.com/redhat-appstudio/o11y/blob/main/rhobs/alerting/data_plane/prometheus.latency_pac_provision_alerts.yaml
Could be running into same troubles as other two mentioned alerts, do you want me to update it also?

**Review by dirgim — APPROVED** [review]
lgtm, but will defer to the o11y team

**Review by amisstea — APPROVED** [review]
LGTM

**Comment by mmorhun** [comment]
@jsztuka we haven't experienced problems so far with PaC metrics, but I think we can increase the value, because 1 minute looks too sensetive

**Review by yftacherzog — APPROVED** [review]

**Pull request merged**